src/fairchem/core/trainers/base_trainer.py

```diff
@@ -358,6 +370,9 @@ def load_datasets(self) -> None:
 
     def load_task(self):
         # Normalizer for the dataset.
+
+        # Is it troublesome that we assume any normalizer info is in train? What if there is no
```

Comment:
I think it is troublesome. Before this PR, it looks like a train dataset was always needed (ie original line 280 `if self.config["dataset"].get("src", None):`, will throw an error if it is not defined.

We should have a conversation of moving this out of the dataset configuration in #715, re this comment https://github.com/FAIR-Chem/fairchem/blob/410f711155b0348cf1f9a8740e03b581b8c1169e/src/fairchem/core/modules/transforms.py#L22-L24

Reply:
I agree that normalizations should not be a property of the dataset!

Reply:
Adding another comment just to trigger a review =)

Reply:
And another

Reply:
And another

Reply:
another